java/src/jmri/jmrit/entryexit/DestinationPoints.java

```diff
@@ -107,7 +107,7 @@ public void propertyChange(PropertyChangeEvent e) {
         };
     }
 
-    String getUniqueId() {
+    public String getUniqueId() {
```

Comment:
This is not needed.  getSystemName() works for the LogixNG store process.

Reply:
Fixed.